automated_analysis.py

```diff
@@ -68,6 +69,27 @@
             individuals[i] = dict(individuals[i].items())
     log.info(f"Loaded {len(individuals)} individuals")
 
+    def coding_plans_to_analysis_configurations(coding_plans):
+        analysis_configurations = []
+        for plan in coding_plans:
+            ccs = plan.coding_configurations
+            for cc in ccs:
+                if not cc.include_in_theme_distribution:
+                    continue
+
+                analysis_configurations.append(
+                    AnalysisConfiguration(cc.analysis_file_key, plan.raw_field, cc.coded_field, cc.code_scheme)
+                )
+        return analysis_configurations
+
+    log.info("Computing engagement counts...")
+    with open(f"{automated_analysis_output_dir}/engagement_counts.csv", "w") as f:
+        engagement_counts.export_engagement_counts_csv(
+            messages, individuals, CONSENT_WITHDRAWN_KEY,
+            coding_plans_to_analysis_configurations(PipelineConfiguration.RQA_CODING_PLANS),
+            f
+        )
+
     # Compute the number of messages, individuals, and relevant messages per episode and overall.
     log.info("Computing the per-episode and per-season engagement counts...")
```

Comment:
You should be able to remove this given the engagement counts export above

Reply:
Thanks forgot this here. Removed.